com.longbridgeapp.LongbridgePro.yaml

```diff
@@ -0,0 +1,81 @@
+app-id: com.longbridgeapp.LongbridgePro
+default-branch: stable
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
```

Comment:
current runtime 23.08

Reply:
nevermind the electron stuff is lagging

Reply:
Ok give it a few hours and 23.08 can be used.

Reply:
> Ok give it a few hours and 23.08 can be used.

Done.